app/Services/Forge/Pipeline/PutCommentOnPullRequest.php

```diff
@@ -14,64 +14,71 @@
 namespace App\Services\Forge\Pipeline;
 
 use App\Services\Forge\ForgeService;
+use App\Services\Github\GithubService;
 use App\Traits\Outputifier;
 use Closure;
-use Illuminate\Support\Facades\Http;
 
 class PutCommentOnPullRequest
 {
     use Outputifier;
 
+    public function __construct(public GithubService $githubService)
+    {
+    }
+
     public function __invoke(ForgeService $service, Closure $next)
     {
-        if ($service->setting->gitCommentEnabled && $service->siteNewlyMade) {
+        if ($service->setting->gitCommentEnabled || ! $service->siteNewlyMade) {
```

Comment:
I am curious to know if the process occurs only if comments are enabled AND the site was created recently